core/math/math_2d.h

```diff
@@ -43,6 +43,14 @@ enum Margin {
 	MARGIN_BOTTOM
 };
 
+enum Corner {
```

Comment:
I though it might could be used some where else so it does not hurt to give access to it by just importing math 2d...
But if you think it is improbable that it will be used seomwhere else I move it to style box